pkg/etc/init.d/wavefront-proxy

```diff
@@ -106,28 +107,36 @@ download_jre() {
             echo "Authenticating as $proxy_user" >&2
             proxy_args+=" --proxy-user $proxy_user:$proxy_password"
         fi
-    else
-        echo "No HTTP proxy configuration detected - attempting direct download" >&2
     fi
-    curl -L --silent -o /tmp/jre.tar.gz $proxy_args https://s3-us-west-2.amazonaws.com/wavefront-misc/proxy-jre-11.0.6-linux_x64.tar.gz || true
+    echo "Downloading and installing JRE $bundled_jre_version" >&2
+    curl -L --silent -o /tmp/jre.tar.gz $proxy_args https://s3-us-west-2.amazonaws.com/wavefront-misc/proxy-jre-$bundled_jre_version-linux_x64.tar.gz || true
     tar -xf /tmp/jre.tar.gz --strip 1 -C $proxy_jre_dir || true
     rm /tmp/jre.tar.gz || true
 }
 
 # If $PROXY_JAVA_HOME is not defined and there is no JRE in $proxy_jre_dir, try to auto-detect
-# locally installed JDK first. We will accept 1.8, 9, 10, 11.
+# locally installed JDK first. We will accept 8, 9, 10, 11.
 if [[ -z "$PROXY_JAVA_HOME" && ! -r $proxy_jre_dir/bin/java ]]; then
-    if type -p java; then
-        JAVA_VERSION=$(java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1)
-        if [[ $JAVA_VERSION == "11" || $JAVA_VERSION == "10" || $JAVA_VERSION == "9" || $JAVA_VERSION == "8" || $JAVA_VERSION == "1.8" ]]; then
-            JAVA_HOME=$($(dirname $(dirname $(readlink -f $(which javac)))))
-            echo "Using Java runtime $JAVA_VERSION detected in $JAVA_HOME" >&2
+    # if java found in path
+    if type -p java > /dev/null 2>&1; then
+        JAVA_VERSION=$(java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1) || true
```

Comment:
i personally prefer awk :)
```java -version 2>&1 | head -1 | awk -F "\"" '{ print $2 }'

Reply:
that command doesn't quite work unfortunately:
``` 
$ java -version 2>&1 | head -1 | awk -F "\"" '{ print $2 }'
1.8.0_202-ea
$ java -version 2>&1 | head -1 | cut -d'"' -f2 | sed '/^1\./s///' | cut -d'.' -f1
8
```

Reply:
and you still need to remove the leading "1." in order to be able to split it by "." afterwards.

Reply:
i see, we can also use `java.specification.version` from `java -XshowSettings:properties`, but i see we want `8` for `1.8` as well.